### ISSUE TITLE
UI Request: Delta Green Color Theming and Logo Alteration

### ISSUE THREAD

**Issue by Agnestorm** [TheLastScrub/delta-green-foundry-vtt-system#153]
The current UI for the system is nice with the altered pause logo and the character sheets, but there's a lot to be desired as a lot of it still uses the default Foundry UI which doesn't really match the rest of the Delta Green vibe. So perhaps it'd be interesting to replace the top-left Foundry anvil icon with the Delta Green logo? And possibly change the highlight shadows and selection border colors of the system from the bland foundry orange to a dark green? (to match the Delta Green style)

**Comment by Agnestorm** [comment]
Just saw that this got closed but no system update yet. Is this being merged soon?

**Comment by TheLastScrub** [comment]
Yes soon, just would like to try to fix the skill sorting issues the non-English locales are facing before I package it all up, and was hoping for one more localization (Polish) to come in as well. I night continue to tweak it a bit, but here is more or less what it looks like though if you are curious:

![Image](https://github.com/user-attachments/assets/c9fd013f-270f-4a6c-b92f-7444a0a042c9)

![Image](https://github.com/user-attachments/assets/81206b2c-a55d-4575-8534-39cd4af154ba)

**Comment by Agnestorm** [comment]
Looking great. Exactly what I imagined it should of looked like.

**Comment by Agnestorm** [comment]
Re-commenting on this a minor request to see if this theming could also be extended to Journal entries and also the change of the scrollbar to a DG styled green to match the rest of the theme too? Thanks
